package.json

```diff
@@ -17,6 +17,7 @@
     "lint": "eslint --ext js,ts,tsx ./",
     "lint:fix": "yarn run lint --fix",
     "gitlint": "node ./internals/scripts/gitlint.js",
+    "docslint": "markdownlint --ignore '**/node_modules/**' '**/*.md'",
```

Comment:
For command name, I would rather use the same style as "build", "build-preview", "build-internal", so:
"lint", "lint-git", "lint-docs".